lib/assets/javascripts/_modules/search.js

```diff
@@ -200,13 +198,23 @@
     function showResults () {
       $searchResultsWrapper.addClass('is-open')
         .attr('aria-hidden', 'false')
+      $searchContainer.attr('aria-modal', 'true')
+      $mainContent.prop('inert', true)
+      $tocNav.prop('inert', true)
+      $header.prop('inert', true)
+      $headerContent.prop('inert', true)
```

Comment:
Given `$headerContent` is inside `$header`, and the `inert` property [affects all descendants](https://whatpr.org/html/4288/interaction.html#the-inert-attribute), I don't think we need to apply the attribute to both?

```suggestion
```